ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -1491,5 +1492,19 @@ public Object Evaluate(String expression)
         }
 
         public String Author { get; set; }
+
+        public List<Drawings.IXLPicture> Pictures()
+        {
+            return pictures;
+        }
+
+        public void AddPicture(Drawings.XLPicture pic)

```

Comment:
Should duplicates really be of ClosedXML's concern? If the user is worried about memory usage they should be taking precautions to avoid duplicates.